tests/unit/src/unit/TestRemoting.hx

```diff
@@ -12,9 +12,9 @@ class TestRemoting extends Test {
 	static var _ : Dynamic = init();
 	static var ecnx : haxe.remoting.ExternalConnection;
 	static var ecnx2 : haxe.remoting.ExternalConnection;
-	static var ecnx3 : haxe.remoting.ExternalConnection;
+	// static var ecnx3 : haxe.remoting.ExternalConnection;
```

Comment:
Oh well, I don't think the remoting tests have been executed this decade anyway.